fiftyone/brain/internal/core/redaction.py

```
@@ -0,0 +1,426 @@
+            out_opts=[
+                "-pix_fmt",
+                "yuv420p",
+                "-c:v",
+                "libopenh264",  # this increases the video quality
+            ],
+        ) as vp:
+            if vp.total_frame_count != len(detections_object_list):
+                raise ValueError(
+                    f"Number of video frames ({vp.total_frame_count}) does not match the number of detections ({len(detections_object_list)})"
+                )
+            for frame, detections_object in zip(vp, detections_object_list):
```

Ruff (0.14.11)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)